docs/source/contributor-guide/testing.md

```diff
@@ -79,8 +79,15 @@ than the standard test suite and add important test coverage such as that the
 code works when there are hash collisions as well as running the relevant
 portions of the entire [sqlite test suite].
 
+You can run the extended tests on any PR by leaving the following comment (see [example here]):
```

Comment:
so to run extended tests, is it optional or is it to rerun failed tests?

Reply:
Normally `extended` tests don't run on PRs, only after merges to main

If you want to run extended tests on a PR prior to merge, now you by leaving a `Run extended test` comment on the PR